src/meta/src/hummock/manager/mod.rs

```diff
@@ -1149,10 +1167,72 @@ where
                 ..Default::default()
             },
         );
+
         let mut new_hummock_version = old_version;
-        new_hummock_version.id = new_version_id;
         new_version_delta.id = new_version_id;
+        new_hummock_version.id = new_version_id;
 
+        for (group_id, is_not_empty) in old_version_groups {
+            if !is_not_empty && !compaction_groups.contains_key(&group_id) {
+                let level_deltas = &mut new_version_delta
+                    .level_deltas
+                    .entry(group_id)
+                    .or_default()
+                    .level_deltas;
+                level_deltas.push(LevelDelta {
+                    delta_type: DeltaType::Groupdestroy as i32,
+                    level_idx: u32::MAX,
+                    l0_sub_level_id: u64::MAX,
+                    ..Default::default()
```

Comment:
Should also set removed_table_ids field, so that group's SSTs can be removed in extend_ssts_to_delete_from_deltas.

Reply:
> Should also set removed_table_ids field, so that group's SSTs can be removed in extend_ssts_to_delete_from_deltas.

No. Currently we remove after SSTs are deleted.

Reply:
> Should also set removed_table_ids field, so that group's SSTs can be removed in extend_ssts_to_delete_from_deltas.

fixed